introduction.md

```diff
@@ -1,4 +1,11 @@
 ##About Eudris Cabrera
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?